scripts/SaveDeployedAddresses.ts

```diff
@@ -1,13 +1,11 @@
-import { promisify } from 'es6-promisify';
```

Comment:
does anything use promisify now? or can we remove it from `package.json`?

Reply:
It's used in SignatureHelper, but I think we can use `util.promisify` instead. Just updated.